apiserver/bundle/bundle_test.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bundle_test
+
+import (
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/apiserver/bundle"
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/params"
+	apiservertesting "github.com/juju/juju/apiserver/testing"
+	statetesting "github.com/juju/juju/state/testing"
+)
+
+type bundleSuite struct {
+	statetesting.StateSuite
+	facade bundle.Bundle
+}
+
+var _ = gc.Suite(&bundleSuite{})
+
+func (s *bundleSuite) SetUpTest(c *gc.C) {
+	s.StateSuite.SetUpTest(c)
+	model, err := s.State.ControllerModel()
+	c.Assert(err, jc.ErrorIsNil)
+	auth := apiservertesting.FakeAuthorizer{
+		Tag: model.Owner(),
```

Comment:
Just set it to some fake user tag. We don't care about the relationship to the model (clearly, since you're not using State), just that it's a client making the request.